src/rt/sched/cown.h

```diff
@@ -967,16 +963,103 @@ namespace verona::rt
       return false;
     }
 
+    inline bool check_unmute_message(Alloc* alloc, MessageBody* msg)
+    {
+      if (msg->behaviour != nullptr)
+        return false;
+
+      for (size_t i = 0; i < msg->count; i++)
+      {
+        auto* cown = msg->cowns[i];
+        if (cown == nullptr)
```

Comment:
why would a cown be null here? would it be first? last? anything in between? multiple?

Reply:
The array of cowns from the MultiMessage that caused their muting is reused in the unmute message. If fewer than the original count of cowns were muted, then the array is null terminated. I'll add a comment in `apply_backpressure` that mentions this.